code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -1538,6 +1538,11 @@ func (sc *StorageSmartContract) finishAllocation(
 	before := make([]currency.Coin, len(sps))
 	deductionFromWritePool := currency.Coin(0)
 
+	challenges, err := sc.getAllocationChallenges(alloc.ID, balances)
+	if err != nil {
+		logging.Logger.Info("finishAllocation: getAllocationChallenges", zap.Error(err))
```

Comment:
Please do not ignore error returned from MPT.

Reply:
In this case, we are getting value not present error for the sc.ID. I am ignoring it because it doesn't matter much if there are no open challenges. 

Not sure, how to deal with this. Can you please guide me on it?

Reply:
Yes, it's okay to ignore the `value not present` error, but it can also be `node not found` error,  which is returned when the miner's state is not fully synced. It's a common error that happens a lot in miners and randomly (especially after restarting). If we see `node not found` error we should return immediately. So the code could be:

```go
if err != nil {
  if err != util.ErrValueNotPresent {
    return fmt.Errorf("could not get allocation challenges: %v", err)
  } 
  log the value not present error if wanted
}

```